libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -290,6 +290,24 @@ public QnAMakerDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNu
         [JsonProperty("rankerType")]
         public StringExpression RankerType { get; set; } = new StringExpression(RankerTypes.DefaultRankerType);
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to enable PreciseAnswer generation. 
+        /// </summary>
+        /// <value>
+        /// A value indicating user's choice between precise answer and full text in response to QnAMaker GenerateAnswer query.
+        /// </value>
+        [JsonProperty("enablePreciseAnswer")]
+        public bool EnablePreciseAnswer { get; set; }
```

Comment:
>bool [](start = 15, length = 4)

Should this be BooleanExpression ?

Reply:
You need to update QnAMakerDialog.schema files if you are adding properties.

---
In reply to: [437801008](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r437801008) [](ancestors = 437801008)

Reply:
Yes. Updated.

---
In reply to: [437801008](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r437801008) [](ancestors = 437801008)